client/header/activity-panel/unread-indicators.js

```diff
@@ -57,3 +65,56 @@ export function getUnreadNotes( select ) {
 export function getLowStockCount() {
 	return getSetting( 'lowStockCount', 0 );
 }
+
+function getIncompleteTasksCount( tasks, dismissedTasks ) {
+	if ( ! tasks ) {
+		return 0;
+	}
+	return tasks.filter(
+		( task ) =>
+			task.visible &&
+			! task.completed &&
+			! dismissedTasks.includes( task.key )
+	).length;
+}
+
+export function isAbbreviatedPanelVisible(
```

Comment:
If we switch to the prop drilling approach mentioned in my comment about the `applyFilters()` usage, I think we can remove this.